components/calendar/calendar.js

```diff
@@ -630,6 +630,12 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		await this._updateFocusDate(date, false, allowDisabled);
 	}
 
+	showYearAndMonth(year, month) {
```

Comment:
Would `nextMonth()` and `previousMonth()` make any sense instead of this? Just so they can avoid doing the calculation of what the next/prev month/year are.

Reply:
Unfortunately MVC Calendar actually needs a way to set the specific year and month because it re-renders the calendar via partial render. But otherwise yeah, this would be preferred.

Reply:
I see. Could it... not do the partial anymore, or is it needed for other things?